Instructions for RN 29?

Environment:

```
react-native --version
react-native-cli: 1.0.0
react-native: 0.29.2
```

After following the instructions for Android, getting the following when running `react-native run-android`

```
Building and installing the app on the device (cd android && ./gradlew installDebug...

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':app'.
> Cannot evaluate module RNSimpleAlertDialogModule : Configuration with name 'default' not found.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED
```

I think the instructions for Android don't match up.  What I mean is it looks like FB changed some things around wrt `MainActivity.java` and 

Here's my initial `MainActivity`:

```
package com.snowflake;

import com.facebook.react.ReactActivity;

public class MainActivity extends ReactActivity {
    /**
     * Returns the name of the main component registered from JavaScript.
     * This is used to schedule rendering of the component.
     */
    @Override
    protected String getMainComponentName() {
        return "snowflake";
    }
}
```

And here's the `MainApplication`:

```
package com.snowflake;

import android.app.Application;
import android.util.Log;

import com.facebook.react.ReactApplication;
import com.facebook.react.ReactInstanceManager;
import com.facebook.react.ReactNativeHost;
import com.facebook.react.ReactPackage;
import com.facebook.react.shell.MainReactPackage;

import java.util.Arrays;
import java.util.List;

public class MainApplication extends Application implements ReactApplication {

  private final ReactNativeHost mReactNativeHost = new ReactNativeHost(this) {
    @Override
    protected boolean getUseDeveloperSupport() {
      return BuildConfig.DEBUG;
    }

    @Override
    protected List<ReactPackage> getPackages() {
      return Arrays.<ReactPackage>asList(
          new MainReactPackage()
      );
    }
  };

  @Override
  public ReactNativeHost getReactNativeHost() {
      return mReactNativeHost;
  }
}
```

So I attempted to modify `MainApplication` w/ the steps documented for `MainActivity` but that results in error as presented above.

NOTE: the import changes wrt React vs React-Native have already been done for RN 25, see [https://github.com/bartonhammond/react-native-simpledialog-android/blob/master/index.js#L7](https://github.com/bartonhammond/react-native-simpledialog-android/blob/master/index.js#L7).  I'd like to get this merged back in if possible.

Thanks in advance!

--- a/README.md
+++ b/README.md
@@ -14,7 +14,7 @@ npm install react-native-simpledialog-android --save
 
 ### Add it to your android project
 
-* In `android/setting.gradle`
+* In `android/settings.gradle`
 
 ```gradle
 ...
@@ -32,12 +32,14 @@ dependencies {
 }
 ```
 
-* Register Module >= 0.17 (in MainActivity.java)
+* Register Module >= 0.17 && <= 0.29(in ```MainActivity.java```)
+* NOTE: >= RN 29 split ```MainActivity.java``` into ```MainActivity.java``` and
+  ```MainApplication.java```.  So make modifications below to ```MainApplication.java```
 
 ```java
 import com.burnweb.rnsimplealertdialog.RNSimpleAlertDialogPackage;  // <--- import
 
-public class MainActivity extends ReactActivity {
+public class MainActivity/MainApplication extends ReactActivity {
   ......
 
   @Override

--- a/index.js
+++ b/index.js
@@ -3,7 +3,7 @@
  */
 
 'use strict';
-
+import React from 'react';
 import { AlertIOS, Platform, NativeModules } from 'react-native';
 
 const SimpleAlertAndroid = NativeModules.SimpleAlertAndroid;

